### PR TITLE
worked on the background for valor and also adjusted the modal style

### PR DESCRIPTION
**Pull request by joshuaolusayo**

### CODE REVIEW

File: valor.css [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 
 namaka-ban::part(container),
 namaka-unban::part(container),
 namaka-edit-comment::part(container),
 namaka-author-reply::part(container),	
 namaka-archive-comment::part(container),
 namaka-board-stats::part(container),
+namaka-comments::part(container),
+namaka-board::part(container),
 namaka-write-a-comment::part(container) {
```
Comment (chase-moskal) [inline valor.css:14]: i don't think we should use this `::part(container)` strategy

if it's possible, i think we should actually try to apply these styles onto the host elements directly, ie

```css
namaka-ban,
namaka-unban
/*...*/ {
  background: #FFAF02;
}
```

let's at least check if this simpler approach is possible

Comment (chase-moskal) [reply valor.css:14]: even though i think we should avoid the `::part(container)` pattern,

it's an example of where we could use `:is` to reduce a little bit of repetition :monocle_face: 

```css
:is(
    namaka-ban,
    namaka-unban,
    namaka-edit-comment,
    namaka-author-reply,
    namaka-archive-comment,
    namaka-board-stats,
    namaka-comments,
    namaka-board,
    namaka-write-a-comment
  )::part(container) {
  background: #FFAF02;
}
```

File: valor.css [hunk 2]
```diff
@@ -48,6 +54,7 @@ namaka-edit-comment::part(pryColor),
 namaka-write-a-comment::part(pryColor) {
```
Comment (chase-moskal) [inline valor.css:54]: i'm not sure what `pry` means in this context?

File: valor.css [hunk 3]
```diff
@@ -129,3 +136,14 @@ namaka-archive-comment::part(bold),
 namaka-ban::part(bold) {
 	color: #000;
 }
+
+namaka-comments {
+	/* border: none; */
+	padding: 0;
+}
+
+namaka-comment::part(btn) {
```
Comment (chase-moskal) [inline valor.css:145]: prefer to use full words like `button` instead of acronyms or shorthands like `btn`

**Review by chase-moskal — CHANGES_REQUESTED** [review]
hello @joshuaolusayo, please see the comments below

File: valor.css (hunk 1 above)
Comment (joshuaolusayo) [reply valor.css]: The second method works and it's been fixed

File: valor.css (hunk 3 above)
Comment (joshuaolusayo) [reply valor.css]: Fixed

**Review by chase-moskal — CHANGES_REQUESTED** [review]
@joshuaolusayo

- [x] i don't think the fonts (`.ttf` files) are legal for us to include.
    - please rewrite the branch to exclude the fonts
- [ ] your branch now has an unwanted merge commit
    - please rewrite the branch to exclude the merge commit

**Comment by joshuaolusayo** [comment]
Issue fixed. I just deleted the font folder

**Pull request merged**